XCUITests/TopTabsTest.swift

```diff
@@ -0,0 +1,162 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import XCTest
+
+let url = "www.mozilla.org"
+let urlLabel = "Internet for people, not profit — Mozilla"
+let urlValue = "mozilla.org"
+
+let urlYah = "www.yahoo.com"
+let urlLabelYah = "Yahoo"
+let urlValueYah = "yahoo"
+
+let urlExample = "example.com"
+
+class TopTabsTest: BaseTestCase {
+    var navigator: Navigator!
+    var app: XCUIApplication!
+
+    override func setUp() {
+        super.setUp()
+        app = XCUIApplication()
+        navigator = createScreenGraph(app).navigator(self)
+    }
+
+    override func tearDown() {
+        super.tearDown()
+    }
+
+    func testAddTabFromSettings() {
+        navigator.goto(NewTabMenu)
+        app.collectionViews.cells["NewTabMenuItem"].tap()
```

Comment:
IMHO, current way is better, since at the script level that we know it created a new tab. I guess the name "NewTabScreen" is kinda misleading for the state since we didn't really create a new tab.  You could rename the existing node as TabScreen for clarity, (but don't create another NewTabScreen, since there is no essential 'state' difference between any tab and a newly created tab.  the 'new tab' best refers to the action of creating a new tab, not its end state.

Yeah, and I doubt you can create multiple edges to the same destination node... hence you cannot define another tap to action to an already defined node I believe.  (I could be wrong)